client/components/date-picker/docs/example.jsx

```diff
@@ -2,122 +2,172 @@
+	},
+
+	{
+		title: 'Social Media - Eventbrite',
+		date: new Date(),
+		socialIcon: 'eventbrite',
+		socialIconColor: false,
+	},
+
+	{
+		title: 'Gridicon - Time',
+		date: new Date(),
+		icon: 'time',
+	},
+
+	{
+		title: 'Gridicon - Offline',
+		date: new Date(),
+		icon: 'offline',
+	},
+
+	{
+		title: 'Gridicon - Shipping',
+		date: new Date(),
+		icon: 'shipping',
+	},
+
+	{
+		title: 'Tomorrow is tomorrow',
+		date: new Date( +new Date() + 60 * 60 * 24 * 1000 ),
+		type: 'future',
+	},
+	{
+		title: 'Yesterday',
+		date: new Date( +new Date() - 60 * 60 * 24 * 1000 ),
+		type: 'past',
+	},
+	{
+		title: 'Retro birthday',
+		date: new Date( '1977-07-18' ),
+		type: 'birthday',
+		icon: 'offline',
+	}
+];
 
 /*
  * Date Picker Demo
  */
 class DatePickerExample extends Component {
 	state = {
-		events: [
-			{
-				title: 'Today',
-				date: new Date(),
-				type: 'scheduled'
-			},
-
-			{
-				title: 'Social Media - Facebook',
-				date: new Date(),
-				socialIcon: 'facebook',
-			},
-
-			{
-				title: 'Social Media - Twitter',
-				date: new Date(),
-				socialIcon: 'twitter',
-				socialIconColor: false,
-			},
-
-			{
-				title: 'Social Media - Google Plus',
-				date: new Date(),
-				socialIcon: 'google-plus',
-			},
-
-			{
-				title: 'Social Media - LinkedIn',
-				date: new Date(),
-				socialIcon: 'linkedin',
-			},
-
-			{
-				title: 'Social Media - Tumblr',
-				date: new Date(),
-				socialIcon: 'tumblr',
-			},
-
-			{
-				title: 'Social Media - Path',
-				date: new Date(),
-				socialIcon: 'path',
-				socialIconColor: false,
-			},
-
-			{
-				title: 'Social Media - Eventbrite',
-				date: new Date(),
-				socialIcon: 'eventbrite',
-				socialIconColor: false,
-			},
-
-			{
-				title: 'Gridicon - Time',
-				date: new Date(),
-				icon: 'time',
-			},
-
-			{
-				title: 'Gridicon - Offline',
-				date: new Date(),
-				icon: 'offline',
-			},
-
-			{
-				title: 'Gridicon - Shipping',
-				date: new Date(),
-				icon: 'shipping',
-			},
-
-			{
-				title: 'Tomorrow is tomorrow',
-				date: new Date( +new Date() + 60 * 60 * 24 * 1000 ),
-				type: 'future-event',
-			},
-			{
-				title: 'Yesterday',
-				date: new Date( +new Date() - 60 * 60 * 24 * 1000 ),
-				type: 'past-event',
-			}
-		]
+		eventsByDay: [],
+		selectedDay: null,
+		showTooltip: false,
 	};
 
 	selectDay = ( date, modifiers ) => {
 		this.setState( { selectedDay: date } );
 
 		if ( date ) {
-			console.log( date.toDate(), modifiers );
+			console.log( date.toDate(), modifiers ); // eslint-disable-line no-console
 		}
 	};
 
+	handleDayMouseEnter = ( date, modifiers, event, eventsByDay ) => {
+		this.setState( {
+			eventsByDay,
+			context: event.target,
+			showTooltip: true,
+		} );
+	};
+
+	handleDayMouseLeave = () => {
+		this.setState( {
+			eventsByDay: [],
+			context: null,
+			showTooltip: false,
+		} );
+	};
+
 	render() {
+		// custom tooltip title
+		const tooltipTitle = this.props.translate(
+			'%d event',
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( '%d Event', '%d Events' )` *ES Score: 25*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).